<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalibrationWidget</class>
 <widget class="QWidget" name="CalibrationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>884</width>
    <height>434</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Calibration</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="6" column="0" colspan="12">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>863</width>
        <height>383</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0">
        <widget class="QGroupBox" name="imageBox">
         <property name="acceptDrops">
          <bool>false</bool>
         </property>
         <property name="title">
          <string>Images</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>4</number>
          </property>
          <property name="leftMargin">
           <number>8</number>
          </property>
          <property name="topMargin">
           <number>8</number>
          </property>
          <property name="rightMargin">
           <number>8</number>
          </property>
          <property name="bottomMargin">
           <number>8</number>
          </property>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="videoBox"/>
   </item>
   <item row="1" column="4">
    <widget class="QLineEdit" name="widthLine">
     <property name="text">
      <string>9</string>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>height</string>
     </property>
    </widget>
   </item>
   <item row="1" column="8">
    <widget class="QPushButton" name="startButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Start Calibration</string>
     </property>
    </widget>
   </item>
   <item row="1" column="7">
    <widget class="QPushButton" name="startRecordingButton">
     <property name="text">
      <string>Start Recording</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="updateButton">
     <property name="text">
      <string>Update</string>
     </property>
    </widget>
   </item>
   <item row="1" column="6">
    <widget class="QLineEdit" name="heightLine">
     <property name="text">
      <string>7</string>
     </property>
    </widget>
   </item>
   <item row="1" column="10">
    <widget class="QPushButton" name="saveCalib">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Save Calibration</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Width</string>
     </property>
    </widget>
   </item>
   <item row="1" column="9">
    <widget class="QPushButton" name="stopShowing">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Stop Showing</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>startRecordingButton</sender>
   <signal>released()</signal>
   <receiver>CalibrationWidget</receiver>
   <slot>startRecording()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>497</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>347</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>updateButton</sender>
   <signal>released()</signal>
   <receiver>CalibrationWidget</receiver>
   <slot>updateVideo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>146</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>347</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>startButton</sender>
   <signal>released()</signal>
   <receiver>CalibrationWidget</receiver>
   <slot>startCalibration()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>620</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>347</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>stopShowing</sender>
   <signal>released()</signal>
   <receiver>CalibrationWidget</receiver>
   <slot>stopShowing()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>731</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>395</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveCalib</sender>
   <signal>released()</signal>
   <receiver>CalibrationWidget</receiver>
   <slot>saveMatrix()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>811</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>441</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addImage()</slot>
  <slot>startCalibration()</slot>
  <slot>startRecording()</slot>
  <slot>updateVideo()</slot>
  <slot>stopShowing()</slot>
  <slot>saveMatrix()</slot>
 </slots>
</ui>
